src/commands/singleCommand.ml

```diff
@@ -66,6 +68,9 @@ let main all weak debug verbose verbose_indent json profile quiet module_
   let config_root = CommandUtils.guess_root (Some(root)) in
```

Comment:
I clobbered all of my symlinks and replaced them with deep copies, and the assert persists.  My environment is a Chromebook with Ubuntu running in a chroot--I'm betting that my environment's the problem.  I'm going to quit chasing this one and hope for the best.